src/datasets/features/audio.py

```diff
@@ -229,7 +228,10 @@ def path_to_bytes(path):
             ],
             type=pa.binary(),
         )
-        path_array = pa.array([None] * len(storage), type=pa.string()) if drop_paths else storage.field("path")
+        path_array = pa.array(
+            [os.path.basename(path) if path is not None else None for path in storage.field("path").to_pylist()],
```

Comment:
I would suggest to store full paths to avoid ambiguities.

Reply:
It may store personal information about the user though, not a big fan

Reply:
okay, agree, users can include full paths in a separate field if needed